src/atoms/toogle/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+
+import * as Styles from './styles';
+import { ToggleProps } from './types';
+
+const Toggle = ({
+  isLabelVisible = false,
+  disabled = false,
+  checked = true,
```

Comment:
```suggestion
  checked = false,
```

I think `false` is a better value for the default state. What do you think?